File: Algorithm/Backjun/bronze5/b2525/b2525.py
# 오븐 시계
# https://www.acmicpc.net/problem/2525

a, b = map(int, input().split())
c = int(input())

totalMinute = a*60 + b + c   # 분의 형태로 바꾸기
totalMinute %= 1440  # 24*60=1440
a = int(totalMinute/60)
b = totalMinute % 60

print(a, b)

File: Algorithm/Backjun/bronze5/b2525/b2525.c
// 오븐 시계
// https://www.acmicpc.net/problem/2525

#include <stdio.h>

int main(void) {
  int a, b, c;
  int totalMinute;

  scanf("%d %d", &a, &b);
  scanf("%d", &c);

  totalMinute = (a*60 + b + c) % 1440;
  a = totalMinute / 60;
  b = totalMinute % 60;

  printf("%d %d", a, b);

  return 0;
}
